kafka/conn.py

```diff
@@ -79,7 +79,7 @@ def _raise_connection_error(self):
 
     def _read_bytes(self, num_bytes):
         bytes_left = num_bytes
-        responses = []
+        result_buffer = six.BytesIO()
```

Comment:
This is what I will be going for: http://stackoverflow.com/a/15964489 We'll have to use `buffer` instead of `memoryview`. I'll try to explore that avenue.